src/components/molecules/Visualizer/Plugin/hooks.ts

```diff
@@ -31,6 +38,10 @@ export default function ({
   widget?: Widget;
   block?: Block;
   pluginProperty?: any;
+  shownPluginModalInfo?: PluginModalInfo;
+  showPluginModal?: (modalInfo?: PluginModalInfo) => void;
```

Comment:
`onPluginModalShow`
And below
`onPluginPopupShow`

Reply:
Updated.